ethers-providers/src/transports/ws.rs

```diff
@@ -401,8 +401,9 @@ where
             // Handle ws messages
             resp = self.ws.next() => match resp {
                 Some(Ok(resp)) => self.handle(resp).await?,
-                // TODO: Log the error?
-                Some(Err(_)) => {},
+                Some(Err(_)) => {
```

Comment:
that makes sense, but let's also log here the error

Reply:
+1 let's log the error and otherwise looks good